docs/building/windows-instructions.md

```diff
@@ -69,7 +69,7 @@ After successfully building, run tests in the Visual Studio Test Explorer window
 
 ### Running tests from the command line
 
-From the root, run `build.cmd` and then `build.cmd -runTests`.
+From the root, run `build.cmd` and then `build.cmd -test`.
```

Comment:
In the section above "Running tests from Visual Studio", can we include that they should select this option:
![image](https://user-images.githubusercontent.com/38739674/101385816-25b4d500-3871-11eb-93e6-7fed13b57c21.png)